mods/LunarCry/mod.xml

```diff
@@ -6,19 +6,21 @@
     <Category>Battle Textures</Category>
     <Description>Lunar Cry - Enemy Texture replacement mod.
 
+        Version 2 - I reconverted images to dds using a better method. Images should be clearer, and less issues with transparencies.
+
```

Comment:
Can you please remove this blank line? Do not open a new PR, just commit the change on the same branch